src/gatsby-ssr.js

```diff
@@ -6,8 +6,8 @@ exports.onRenderBody = ({ setPostBodyComponents }, options = {}) => {
 	options = Object.assign({
 		apiKey: process.env.GATSBY_SNIPCART_API_KEY,
 		autopop: false,
-		js: 'https://cdn.snipcart.com/themes/v3.0.22/default/snipcart.js',
-		styles: 'https://cdn.snipcart.com/themes/v3.0.22/default/snipcart.css',
+		js: 'https://cdn.snipcart.com/themes/v3.0.28/default/snipcart.js',
```

Comment:
Worth considering bumping this up to v3.0.31 before merge (if it gets approved at some point 🙂  )

Reply:
Merging mate, apologies I took some time on this!